tests/test_event_optimize.py

```diff
@@ -36,3 +39,48 @@ def test_result(tmp_path):
     finally:
         os.chdir(p)
         sys.stdout = saved_stdout
+
+
+def test_parallel(tmp_path):
+    parfile = datadir / "PSRJ0030+0451_psrcat.par"
+    eventfile_orig = (
+        datadir
+        / "J0030+0451_P8_15.0deg_239557517_458611204_ft1weights_GEO_wt.gt.0.4.fits"
+    )
+    temfile = datadir / "templateJ0030.3gauss"
+    eventfile = tmp_path / "event.fits"
+    # We will write a pickle next to this file, let's make sure it's not under tests/
+    shutil.copy(eventfile_orig, eventfile)
+
+    p = Path.cwd()
+    saved_stdout, sys.stdout = (sys.stdout, StringIO("_"))
+
+    np.random.seed(1)
+    event_optimize.maxpost = -9e99
+    event_optimize.numcalls = 0
+    try:
+        import pathos
+
+        os.chdir(tmp_path)
+        cmd = f"{eventfile} {parfile} {temfile} --weightcol=PSRJ0030+0451 --minWeight=0.9 --nwalkers=10 --nsteps=50 --burnin 10"
+        event_optimize.main(cmd.split())
+        with open("J0030+0451_samples.pickle", "rb") as f:
+            samples1 = pickle.load(f)
+        f.close
+        np.random.seed(1)
+        event_optimize.maxpost = -9e99
+        event_optimize.numcalls = 0
+
+        cmd = f"{eventfile} {parfile} {temfile} --weightcol=PSRJ0030+0451 --minWeight=0.9 --nwalkers=10 --nsteps=50 --burnin 10 --multicores"
+        event_optimize.main(cmd.split())
+        with open("J0030+0451_samples.pickle", "rb") as f:
+            samples2 = pickle.load(f)
+        f.close()
+
+        for i in range(0, samples1.shape[1]):
+            assert stats.ks_2samp(samples1[:, i], samples2[:, i])[1] == 1.0
+    except ImportError:
+        pass
```

Comment:
`pytest.skip` would be appropriate here - indicate that the test is being skipped because `pathos` is not available.